kubeflow/core/jupyterhub.libsonnet

```diff
@@ -1,7 +1,4 @@
 {
-  // TODO(https://github.com/ksonnet/ksonnet/issues/222): Taking namespace as an argument is a work around for the fact that ksonnet
-  // doesn't support automatically piping in the namespace from the environment to prototypes.
-  //
```

Comment:
Isn't this TODO for removing namespace param in parts(namespace)?

Reply:
The comment in the TODO mentions that `Taking namespace as an argument is a work around for the fact that ksonnet doesn't support automatically piping in the namespace from the environment to prototypes` - I don't think that is true. 

The workaround for ksonnet not piping the namespace from env to prototypes is we have an explicit parameter called `namespace` in our prototype - this PR addresses this issue by removing the `namespace` parameter from our prototypes.